web/src/containers/App/AppInner/AppInner.tsx

```diff
@@ -54,11 +55,30 @@ const AppInner = (props: Props) => {
     }
 
     const [initializedByAnother, setInitializedByAnother] = useState(false)
+    const [suspectedUnauthorized, setSuspectedUnauthorized] = useState(false)
+
+    const unauthorizedCallback = () => {
+        setSuspectedUnauthorized(true)
+
+        reFetchConfig().then((newWellKnownConfig: WellKnownConfigType) => {
+            if (authProviderRef) {
+                const userData: User = authProviderRef?.current?.getUserData()
+                const parsedData = jwtDecode(userData.access_token)
+                const ownerId = get(parsedData, 'owner-id', '')
```

Comment:
instead `owner-id` you need to use `wellKnownConfig.remoteProvisioning.authorization.ownerClaim` property

Reply:
so .. no `access_token` but  **wellKnownConfig.remoteProvisioning.authorization.ownerClaim** compare with  **wellKnownConfig.owner**

Reply:
No. You need value from access token, but you hardcoded in which claim you find the value. It doesn't have to be necessary `owner-id`. The claim key should be parametrized, with the value from the path Jozef described.

Reply:
Example
`const ownerId = get(parsedData, wellKnownConfig.remoteProvisioning.authorization.ownerClaim, '')`

Reply:
Oh okay, it was not written down (in task) that it can be something other that 'owner' property. So i get it now. ✅